modules/files/files.go

```diff
@@ -72,17 +99,42 @@ func CopyTerragruntFolderToTemp(folderPath string, tempFolderPrefix string) (str
 		return !PathContainsHiddenFileOrFolder(path) && !PathContainsTerraformState(path)
 	}
 
-	destFolder, err := CopyFolderToTemp(folderPath, tempFolderPrefix, filter)
+	destRootFolder := os.TempDir()
+	destFolder, err := CopyFolderToDest(folderPath, destRootFolder, tempFolderPrefix, filter)
 	if err != nil {
 		return "", err
 	}
 
 	return destFolder, nil
 }
 
-// CopyFolderToTemp creates a copy of the given folder and all its filtered contents in a temp folder
```

Comment:
Ah we shouldn't remove `CopyFolderToTemp`, since it's a public API function, and downstream users have use for that. This should instead be converted to a thin wrapper around `CopyFolderToDest`.

Reply:
Thank you, that's a great point.  I have added CopyFolderToTemp back as a wrapper for CopyFolderToDest using the same form demonstrated in your other comment.